packages/helpers/scripts/ast_grep.js

```diff
@@ -86,27 +86,14 @@ export function ast_grep() {
 
                     source.prependLeft(range.start.index, `exports._ = exports.${func_name} = `);
                 });
-
-            const export_shortname = `export { ${func_name} as _}`;
-
-            const export_alias = tree.root().find(export_shortname);
-
-            if (!export_alias) {
-                task_queue.push(
-                    fs.appendFile(tree.filename(), export_shortname, "utf-8"),
-                );
```

Comment:
@magic-akari Am I good to omit this as well?

```js
task_queue.push(	
  fs.appendFile(tree.filename(), export_shortname, "utf-8"),	
);
```

Reply:
It's OK.